CHANGELOG.md

```diff
@@ -114,12 +114,14 @@
 - Fix descriptive labels on action links
 - Remove `reference` from Transactions
 - Add level D activities (third-party projects)
-<<<<<<< HEAD
```

Comment:
I have no idea what that is doing in there 😱

Reply:
oopsie! That would be me…